test/controllers/close_reasons_controller_test.rb

```diff
@@ -0,0 +1,9 @@
+require 'test_helper'
+
+class CloseReasonsControllerTest < ActionDispatch::IntegrationTest
+  test "should get index" do
+    get close_reasons_index_url
+    assert_response :success
+  end
+
```

Comment:
Could do with some more tests for the other actions - take a look around at how other tests test more complex actions.